fixing CNS 11h to generate identify per allocated namespace.

HaroPanosyan opened the pull request:
Fixing generation of CNS 11h identify namespace data structure generation for each allocated namespace instead of attached namespace.

HaroPanosyan commented:
> Fixing generation of CNS 11h identify namespace data structure generation for each allocated namespace instead of attached namespace.

Please ignore this mess. It would be nice if I can remove this Pull request and then will create new one clean.

igaw commented:
Deleting pull request is not so easy. Only the plattform admins can do this. But it's not that bad you can fix your branch locally with all sort of git operation (rebasing, squashing etc) and just force push it to this branch. This will fix it.

HaroPanosyan commented:
> Deleting pull request is not so easy. Only the plattform admins can do this. But it's not that bad you can fix your branch locally with all sort of git operation (rebasing, squashing etc) and just force push it to this branch. This will fix it.

Thanks, will do, I have a good help here.

HaroPanosyan commented:
Cleaned up the mess with 3 commits. Now only one clean commit to be reviewed.

igaw commented:
Looks good. Just your Signed-off-by tag is missing. I can add it myself if you are okay with it.

HaroPanosyan commented:
> Looks good. Just your Signed-off-by tag is missing. I can add it myself if you are okay with it.

Thanks for your help... I rebased and added Signed-off-by tag.

igaw commented:
Thanks!
